sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusScaleMonitor.cs

```diff
@@ -146,9 +151,19 @@ private ScaleStatus GetScaleStatusCore(int workerCount, ServiceBusTriggerMetrics
                         (prev, next) => prev.QueueTime <= next.QueueTime);
                 if (queueTimeIncreasing)
                 {
-                    status.Vote = ScaleVote.ScaleOut;
-                    _logger.LogInformation($"Queue time is increasing for '{_entityPath}'.");
-                    return status;
+                    // to avoid frequent scaling out when queue time is increasing, we wait for atleast 2s (backward compatibility with SCV2).
+                    if (lastSampleQueueTime >= TimeSpan.FromSeconds(MinimumLastQueueMessageInSecondsThreshold))
```

Comment:
Did we catch the issue for the customer without management permissions for the connection? By default, TargetBaseScaler should be used.

Reply:
This customer was not able to use TargetBasedScaler, they had some permissions issues in their SB connection string. Eventually that was how the ICM was resolved, they fixed the permissions issue and targetBased scaling kicked in to resolve it. The fix here is for all the cases where the cx can not or will not fix their permissions forcing them to use the ScaleMonitor Based scaling.